content/knowledge-base/lifecycle.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Product Lifecycle
+---
+
+## Software List
+
+Upbound provides support and sofwtare maintenance to paid customers. Our supported software products include:
+
+* [UXP](https://www.upbound.io/product/universal-crossplane)
+* [Official Providers](https://marketplace.upbound.io/providers?tier=official)
```

Comment:
**question (non-blocking)**: Would it be better to link to the Upbound docs pages here rather than the external links? 

I.e., https://docs.upbound.io/upbound-marketplace/providers/ and https://docs.upbound.io/uxp/